tests/approaches/test_interactive_approach.py

```diff
@@ -119,3 +119,10 @@ def test_interactive_learning_approach():
     with pytest.raises(NotImplementedError) as e:
         approach._score_atom_set(set(), train_tasks[0].init)  # pylint:disable=protected-access
     assert "Unrecognized interactive_score_function" in str(e)
+    # Test assertion that all predicates are seen in the data
+    utils.update_config({
+        "approach": "interactive_learning",
```

Comment:
idk if my brain's just not working, but this test wouldn't work without this line and i have no idea why

Reply:
yeah this is our fault I think 😬 I can explain what's going on if you're curious. but TLDR, this is good, and in general, `reset_config` is much preferred over `update_config` for tests now